servers/rendering/shader_language.h

```diff
@@ -1011,7 +1011,8 @@ class ShaderLanguage {
 
 	Token _make_token(TokenType p_type, const StringName &p_text = StringName());
 	Token _get_token();
-	bool _lookup_next(Token &r_tk);
+	Token _peek_token();
+	bool _lookup_next_for_completion(Token &r_tk);
```

Comment:
Why this rename?

Reply:
Because it's not clear from the original name, or the (lack of 😁) documentation comment for that function that it peeks two tokens and only returns `true` (and the second token) if it's on the same physical line where the parser had been before calling it.

Naively, I tried `_lookup_next()` first before I realized it's not semantically `_peek_token()` at all.